### PR TITLE
Dala 4695 qa report eu taxo non financials

### PR DESCRIPTION
**Pull request by jakobkraus**
# Pull Request \<Title>
`<Description here>`
## Things to do during Peer Review
Please check all boxes before the Pull Request is merged. In case you skip a box, describe in the PRs description (that means: here) why the check is skipped.
- [x] The Github Actions (including Sonarqube Gateway and Lint Checks) are green. This is enforced by Github. 
- [x] A peer-review has been executed
  - [x] The code has been manually inspected by someone who did not implement the feature
  - [x] If this PR includes work on the frontend, at least one `@ts-nocheck` is removed. Additionally, there should not be any `@ts-nocheck` in files modified by this PR. If no `@ts-nocheck` are left: Celebrate :tada: :confetti_ball: type-safety and remove this entry. 
- [x] The PR actually implements what is described in the JIRA-Issue
- [x] At least one test exists testing the new feature
  - [ ] If you have created new test files, make sure that they are included in a test container and actually run in the CI
- [x] At least 3 consecutive CI runs are successfully executed to ensure that there are no flaky tests.
- [ ] Documentation is updated as required
- [ ] The automated deployment is updated if required
- [ ] If there was a database entity class added, there must also be a migration script for creating the corresponding database if flyway is already used by the service
- [ ] IF there are changes done to the framework data models or to a database entity class, the following steps were completed in order
  - [ ] A fresh clone of dataland.com is generated (see Wiki page on "OTC" for details)
  - [ ] The feature branch is deployed to clone with `Reset non-user related Docker Volumes & Re-populate` turned off
  - [ ] It's verified that the CD run is green
  - [ ] The new feature is manually used/tested/observed on the clone server. Testing of the new feature should (also) be done by a second, independent reviewer from the dev team
  - [ ] The feature branch is deployed to dev1 with `Reset non-user related Docker Volumes & Re-populate` turned on, and it's verified that the CD run is green  
- [x] ELSE, the new version is deployed to the dev server "dev1" using this branch
  - [x] Run with setting `Reset non-user related Docker Volumes & Re-populate` turned on 
  - [x] It's verified that this version actually is the one deployed (check gitinfo for branch name and commit id!)
  - [x] It's verified that the CD run is green
  - [ ] The new feature is manually used/tested/observed on dev server. Testing of the new feature should (also) be done by a second, independent reviewer from the dev team

### CODE REVIEW

File: dataland-framework-toolbox/inputs/euTaxonomyNonFinancials/EutaxonomyNonFinancials.csv
Comment (dennis-dfine) [inline dataland-framework-toolbox/inputs/euTaxonomyNonFinancials/EutaxonomyNonFinancials.csv:1]: What happened here? Some field gained extended document support, and amountWithCurrency was changed to Currency? Why?

File: dataland-framework-toolbox/src/main/kotlin/org/dataland/frameworktoolbox/frameworks/eutaxonomynonfinancials/EuTaxonomyNonFinancialsFramework.kt [hunk 1]
```diff
@@ -23,7 +23,7 @@ class EuTaxonomyNonFinancialsFramework : PavedRoadFramework(
     ),
     order = 2,
     enabledFeatures =
-    FrameworkGenerationFeatures.allExcept(FrameworkGenerationFeatures.UploadPage, FrameworkGenerationFeatures.QaModel),
+    FrameworkGenerationFeatures.allExcept(FrameworkGenerationFeatures.UploadPage),
```
Comment (dennis-dfine) [inline dataland-framework-toolbox/src/main/kotlin/org/dataland/frameworktoolbox/frameworks/eutaxonomynonfinancials/EuTaxonomyNonFinancialsFramework.kt:26]: why was this deleted?

File: dataland-framework-toolbox/src/main/kotlin/org/dataland/frameworktoolbox/frameworks/eutaxonomynonfinancials/custom/EuTaxonomyAlignedActivitiesComponent.kt [hunk 2]
```diff
@@ -49,25 +49,53 @@ class EuTaxonomyAlignedActivitiesComponent(
...
                 listOf(
                     TypeReference(
-                        "org.dataland.datalandbackend.frameworks.eutaxonomynonfinancials" +
-                            ".custom.EuTaxonomyAlignedActivity",
-                        false,
+                        "kotlin.collections.MutableList",
+                        isNullable,
+
+                        listOf(
+
```
Comment (dennis-dfine) [inline dataland-framework-toolbox/src/main/kotlin/org/dataland/frameworktoolbox/frameworks/eutaxonomynonfinancials/custom/EuTaxonomyAlignedActivitiesComponent.kt:60]: empty line

File: dataland-framework-toolbox/src/main/kotlin/org/dataland/frameworktoolbox/frameworks/eutaxonomynonfinancials/custom/EuTaxonomyAlignedActivitiesComponent.kt [hunk 3]
```diff
@@ -49,25 +49,53 @@ class EuTaxonomyAlignedActivitiesComponent(
...
+                "org.dataland.datalandbackend.model.datapoints.ExtendedDataPoint",
                 isNullable,
                 listOf(
                     TypeReference(
-                        "org.dataland.datalandbackend.frameworks.eutaxonomynonfinancials" +
-                            ".custom.EuTaxonomyAlignedActivity",
-                        false,
+                        "kotlin.collections.MutableList",
+                        isNullable,
+
```
Comment (dennis-dfine) [inline dataland-framework-toolbox/src/main/kotlin/org/dataland/frameworktoolbox/frameworks/eutaxonomynonfinancials/custom/EuTaxonomyAlignedActivitiesComponent.kt:58]: empty line

File: dataland-framework-toolbox/src/main/kotlin/org/dataland/frameworktoolbox/frameworks/eutaxonomynonfinancials/custom/EuTaxonomyAlignedActivitiesComponent.kt [hunk 4]
```diff
@@ -49,25 +49,53 @@ class EuTaxonomyAlignedActivitiesComponent(
...
+            TypeReference(
+                "org.dataland.datalandqaservice.model.reports.QaReportDataPoint",
+                isNullable,
+                listOf(
+                    TypeReference(
+                        "org.dataland.datalandbackend.openApiClient.model" +
+                            ".ExtendedDataPointListEuTaxonomyAlignedActivity",
+                        isNullable,
+                    ),
+
```
Comment (dennis-dfine) [inline dataland-framework-toolbox/src/main/kotlin/org/dataland/frameworktoolbox/frameworks/eutaxonomynonfinancials/custom/EuTaxonomyAlignedActivitiesComponent.kt:85]: empty line

File: dataland-framework-toolbox/src/main/kotlin/org/dataland/frameworktoolbox/frameworks/eutaxonomynonfinancials/custom/EuTaxonomyNonAlignedActivitiesComponent.kt [hunk 5]
```diff
@@ -26,19 +26,43 @@ class EuTaxonomyNonAlignedActivitiesComponent(
...
+            TypeReference(
+                "org.dataland.datalandqaservice.model.reports.QaReportDataPoint",
+                isNullable,
+                listOf(
+                    TypeReference(
+                        "org.dataland.datalandbackend.openApiClient.model" +
+                            ".ExtendedDataPointListEuTaxonomyActivity",
+                        isNullable,
+                    ),
+
```
Comment (dennis-dfine) [inline dataland-framework-toolbox/src/main/kotlin/org/dataland/frameworktoolbox/frameworks/eutaxonomynonfinancials/custom/EuTaxonomyNonAlignedActivitiesComponent.kt:60]: empty line

File: dataland-frontend/src/components/forms/CreateHeimathafenDataset.vue [hunk 6]
```diff
@@ -301,8 +299,8 @@ export default defineComponent({
         this.uploadSucceded = true;
       } catch (error) {
         console.error(error);
-        if (error.message) {
-          this.message = formatAxiosErrorMessage(error as Error);
+        if (error instanceof Error) {
```
Comment (dennis-dfine) [inline dataland-frontend/src/components/forms/CreateHeimathafenDataset.vue:302]: 👍

File: dataland-frontend/src/components/forms/parts/fields/CurrencyDataPointFormField.vue [hunk 7]
```diff
@@ -27,16 +28,15 @@
...
           input-class="long"
+          data-test="currency"
         />
       </div>
     </div>
   </ExtendedDataPointFormField>
 </template>
 
 <script lang="ts">
-// @ts-nocheck
```
Comment (dennis-dfine) [inline dataland-frontend/src/components/forms/parts/fields/CurrencyDataPointFormField.vue:37]: 💯

File: dataland-frontend/src/components/forms/parts/fields/PercentageFormField.vue [hunk 8]
```diff
@@ -20,30 +20,33 @@
 </template>
 
 <script lang="ts">
-// @ts-nocheck
```
Comment (dennis-dfine) [inline dataland-frontend/src/components/forms/parts/fields/PercentageFormField.vue:23]: 🥳

File: dataland-frontend/src/frameworks/esg-questionnaire/ViewConfig.ts
Comment (dennis-dfine) [inline dataland-frontend/src/frameworks/esg-questionnaire/ViewConfig.ts:1]: This file is looooooong!

File: dataland-frontend/src/frameworks/eutaxonomy-non-financials/ViewConfig.ts
Comment (dennis-dfine) [inline dataland-frontend/src/frameworks/eutaxonomy-non-financials/ViewConfig.ts:1]: This file is loooong!

File: dataland-frontend/src/frameworks/lksg/ViewConfig.ts [hunk 9]
```diff
@@ -97,19 +97,20 @@ export const lksgViewConfiguration: MLDTConfig<LksgData> = [
...
-                  : ''
-              );
-            },
+            valueGetter: (dataset: LksgData): AvailableMLDTDisplayObjectTypes =>
+              ((): AvailableMLDTDisplayObjectTypes => {
+                const mappings = {
+                  Smaller10: '<10%',
+                  Between10And25: '10-25%',
+                  Between25And50: '25-50%',
+                  Greater50: '>50%',
```
Comment (dennis-dfine) [inline dataland-frontend/src/frameworks/lksg/ViewConfig.ts:106]: not your issue but: what do i choose in case of 25%?

File: dataland-frontend/tests/component/components/forms/CreateEuTaxonomyForNonFinancials.cy.ts [hunk 10]
```diff
@@ -1,4 +1,3 @@
-// @ts-nocheck
```
Comment (dennis-dfine) [inline dataland-frontend/tests/component/components/forms/CreateEuTaxonomyForNonFinancials.cy.ts:1]: 👏

File: gitinfo
Comment (dennis-dfine) [inline gitinfo:1]: remove file

File: dataland-internal-storage/src/main/kotlin/db/migration/V19__MigrateEutaxonomyNonFinancialsExtendedDatapoints.kt [hunk 11]
```diff
@@ -0,0 +1,117 @@
...
+        jsonObject.keys().forEach {
+            if (it == "absoluteShare") {
+                val absoluteShare = jsonObject[it] as JSONObject
+                val amount = absoluteShare["amount"]
+                absoluteShare.remove("amount")
+                absoluteShare.put("value", amount)
+            } else if (it in relevantFields) {
+                createNestedJsonObject(jsonObject, it)
+            } else {
+                // Do nothing
```
Comment (dennis-dfine) [inline dataland-internal-storage/src/main/kotlin/db/migration/V19__MigrateEutaxonomyNonFinancialsExtendedDatapoints.kt:63]: remove `else` ?

File: dataland-internal-storage/src/main/kotlin/db/migration/V19__MigrateEutaxonomyNonFinancialsExtendedDatapoints.kt [hunk 12]
```diff
@@ -0,0 +1,117 @@
...
+            checkRecursivelyForBaseDataPointsInJsonObject(jsonObject, it)
+        }
+    }
+
+    /**
+     * Check recursively for BaseDataPoints in a JSON array.
+     * @param jsonArray JSON array
+     */
+    private fun checkRecursivelyForBaseDataPointsInJsonArray(jsonArray: JSONArray) {
+        for (i in 0 until jsonArray.length()) {
```
Comment (dennis-dfine) [inline dataland-internal-storage/src/main/kotlin/db/migration/V19__MigrateEutaxonomyNonFinancialsExtendedDatapoints.kt:74]:
```suggestion
        for (element in jsonArray) {
```

File: dataland-internal-storage/src/main/kotlin/db/migration/V19__MigrateEutaxonomyNonFinancialsExtendedDatapoints.kt [hunk 13]
```diff
@@ -0,0 +1,117 @@
...
+     * @param key key corresponding to the JSON object
+     */
+    private fun checkRecursivelyForBaseDataPointsInJsonObject(jsonObject: JSONObject, key: String) {
+        val obj = jsonObject.getOrJavaNull(key)
+        if (obj !== null && obj is JSONObject) {
+            checkForRelevantFieldsInJsonObjectKeys(obj)
+        } else if (obj != null && obj is JSONArray) {
+            checkRecursivelyForBaseDataPointsInJsonArray(obj)
+        } else {
+            // Do nothing
```
Comment (dennis-dfine) [inline dataland-internal-storage/src/main/kotlin/db/migration/V19__MigrateEutaxonomyNonFinancialsExtendedDatapoints.kt:94]: remove

File: dataland-framework-toolbox/src/main/kotlin/org/dataland/frameworktoolbox/frameworks/eutaxonomynonfinancials/custom/EuTaxonomyNonAlignedActivitiesComponent.kt (hunk 5 above)
Comment (jakobkraus) [reply dataland-framework-toolbox/src/main/kotlin/org/dataland/frameworktoolbox/frameworks/eutaxonomynonfinancials/custom/EuTaxonomyNonAlignedActivitiesComponent.kt:60]: deleted line

File: dataland-framework-toolbox/src/main/kotlin/org/dataland/frameworktoolbox/frameworks/eutaxonomynonfinancials/custom/EuTaxonomyAlignedActivitiesComponent.kt (hunk 4 above)
Comment (jakobkraus) [reply dataland-framework-toolbox/src/main/kotlin/org/dataland/frameworktoolbox/frameworks/eutaxonomynonfinancials/custom/EuTaxonomyAlignedActivitiesComponent.kt:85]: deleted line

File: dataland-framework-toolbox/src/main/kotlin/org/dataland/frameworktoolbox/frameworks/eutaxonomynonfinancials/custom/EuTaxonomyAlignedActivitiesComponent.kt (hunk 3 above)
Comment (jakobkraus) [reply dataland-framework-toolbox/src/main/kotlin/org/dataland/frameworktoolbox/frameworks/eutaxonomynonfinancials/custom/EuTaxonomyAlignedActivitiesComponent.kt]: deleted line

File: dataland-e2etests/src/test/kotlin/org/dataland/e2etests/tests/QaReportControllerTest.kt [hunk 14]
```diff
@@ -39,31 +43,71 @@ class QaReportControllerTest {
...
         val dataIdOfUpload = uploadInfo.first().actualStoredDataMetaInfo!!.dataId
         return withTechnicalUser(TechnicalUser.Admin) {
-            qaApiAccessor.sfdrQaReportControllerApi.postQaReport(dataIdOfUpload, sfdrQaReport)
+            qaApiAccessor.sfdrQaReportControllerApi.postSfdrDataQaReport(dataIdOfUpload, sfdrQaReport)
         }
     }
 
-    private fun assertCanAccessQaDataset(reportMetaInformation: QaReportMetaInformation) {
+    private fun postEuTaxonomyNonFinancialsQaReportForNewDataId(
+        euTaxonomyNonFinancialsQaReport: EuTaxonomyNonFinancialsReport = EuTaxonomyNonFinancialsReport(),
```
[the remark this answers is not in the document]
Comment (jakobkraus) [reply dataland-e2etests/src/test/kotlin/org/dataland/e2etests/tests/QaReportControllerTest.kt:59]: Done

File: dataland-e2etests/src/test/kotlin/org/dataland/e2etests/tests/QaReportControllerTest.kt
[the remark this answers is not in the document]
Comment (jakobkraus) [reply dataland-e2etests/src/test/kotlin/org/dataland/e2etests/tests/QaReportControllerTest.kt:1]: Due to the structure of the code, a rewriting of the private functions would amount to one big if/else (or when) construct per generic function with the same similar-looking code snippets inside the cases.

File: dataland-e2etests/src/test/kotlin/org/dataland/e2etests/utils/testDataProvivders/QaReportTestDataProvider.kt [hunk 15]
```diff
@@ -5,13 +5,25 @@ import com.squareup.moshi.Moshi
 import com.squareup.moshi.Types
 import com.squareup.moshi.kotlin.reflect.KotlinJsonAdapterFactory
 import java.io.File
+import org.dataland.datalandqaservice.openApiClient.model.EutaxonomyNonFinancialsData as EuTaxonomyNonFinancialsQaReport
```
[the remark this answers is not in the document]
Comment (jakobkraus) [reply dataland-e2etests/src/test/kotlin/org/dataland/e2etests/utils/testDataProvivders/QaReportTestDataProvider.kt:8]: Removed as clause

File: dataland-framework-toolbox/src/main/kotlin/org/dataland/frameworktoolbox/frameworks/eutaxonomynonfinancials/EuTaxonomyNonFinancialsFramework.kt (hunk 1 above)
Comment (LaurinStrelow) [reply dataland-framework-toolbox/src/main/kotlin/org/dataland/frameworktoolbox/frameworks/eutaxonomynonfinancials/EuTaxonomyNonFinancialsFramework.kt:26]: Otherwise, the toolbox does not create the qa model

File: dataland-e2etests/src/test/kotlin/org/dataland/e2etests/tests/QaReportControllerTest.kt [hunk 16]
```diff
@@ -26,8 +26,12 @@ class QaReportControllerTest {
     private val qaApiAccessor = QaApiAccessor()
 
     private val testSfdrDataset = apiAccessor.testDataProviderForSfdrData.getTData(1)
-    private val testCompanyInformation = apiAccessor.testDataProviderForSfdrData
+    private val testEuTaxonomyNonFinancialsDataset =
+        apiAccessor.testDataProviderForEuTaxonomyDataForNonFinancials.getTData(1)
+    private val testCompanyInformationSfdr = apiAccessor.testDataProviderForSfdrData
```
[the remark this answers is not in the document]
Comment (jakobkraus) [reply dataland-e2etests/src/test/kotlin/org/dataland/e2etests/tests/QaReportControllerTest.kt]: Made name more accessible

File: dataland-e2etests/src/test/kotlin/org/dataland/e2etests/utils/testDataProvivders/QaReportTestDataProvider.kt [hunk 17]
```diff
@@ -5,13 +5,25 @@ import com.squareup.moshi.Moshi
...
 import org.dataland.datalandqaservice.openApiClient.model.SfdrData as SfdrQaReport
 
-class QaReportTestDataProvider<T> (private val clazz: Class<T>) {
+class QaReportTestDataProvider<T>(private val clazz: Class<T>, private val framework: String) {
 
-    private val jsonFilesForTesting = mapOf(
+    private val sfdrJsonFilesForTesting = mapOf(
         SfdrQaReport::class.java to File("./build/resources/test/SfdrQaReportPreparedFixtures.json"),
         /*
+The following line of code is a dummy entry to avoid a compilation error.
```
[the remark this answers is not in the document]
Comment (jakobkraus) [reply dataland-e2etests/src/test/kotlin/org/dataland/e2etests/utils/testDataProvivders/QaReportTestDataProvider.kt]: Fixed the indentation

File: dataland-framework-toolbox/src/main/kotlin/org/dataland/frameworktoolbox/frameworks/eutaxonomynonfinancials/custom/EuTaxonomyAlignedActivitiesComponent.kt (hunk 2 above)
Comment (jakobkraus) [reply dataland-framework-toolbox/src/main/kotlin/org/dataland/frameworktoolbox/frameworks/eutaxonomynonfinancials/custom/EuTaxonomyAlignedActivitiesComponent.kt]: deleted line

File: gitinfo
Comment (jakobkraus) [reply gitinfo:1]: Fixed

File: dataland-e2etests/src/test/kotlin/org/dataland/e2etests/tests/MetaDataControllerTest.kt [hunk 18]
```diff
@@ -189,7 +189,7 @@ class MetaDataControllerTest {
                 activeDatasets[0].dataId,
             )
         assertTrue(
-            (retrievedDataset.data.general!!.numberOfEmployees == newNumberOfEmployees),
+            (retrievedDataset.data.general!!.numberOfEmployees!!.value == newNumberOfEmployees),
```
[the remark this answers is not in the document]
Comment (jakobkraus) [reply dataland-e2etests/src/test/kotlin/org/dataland/e2etests/tests/MetaDataControllerTest.kt]: Is marked as a change because I did not merge main into branch earlier

File: dataland-e2etests/src/test/kotlin/org/dataland/e2etests/tests/MetaDataControllerTest.kt [hunk 19]
```diff
@@ -206,9 +206,14 @@ class MetaDataControllerTest {
             reportingPeriod = reportingPeriod,
             uploadFunction = apiAccessor::euTaxonomyNonFinancialsUploaderFunction,
         )
-        val newNumberOfEmployees = (frameworkDataAlpha.general!!.numberOfEmployees ?: BigDecimal.ZERO) + BigDecimal.ONE
+        val newNumberOfEmployees = (frameworkDataAlpha.general!!.numberOfEmployees!!.value ?: BigDecimal.ZERO) +
+            BigDecimal.ONE
         val frameworkDataBeta = frameworkDataAlpha.copy(
-            general = frameworkDataAlpha.general!!.copy(numberOfEmployees = newNumberOfEmployees),
+            general = frameworkDataAlpha.general!!.copy(
+                numberOfEmployees = frameworkDataAlpha.general!!.numberOfEmployees!!.copy(
```
[the remark this answers is not in the document]
Comment (jakobkraus) [reply dataland-e2etests/src/test/kotlin/org/dataland/e2etests/tests/MetaDataControllerTest.kt]: Is marked as a change because I did not merge main into branch earlier

File: dataland-e2etests/src/test/kotlin/org/dataland/e2etests/tests/MetaDataControllerTest.kt [hunk 20]
```diff
@@ -206,9 +206,14 @@ class MetaDataControllerTest {
             reportingPeriod = reportingPeriod,
             uploadFunction = apiAccessor::euTaxonomyNonFinancialsUploaderFunction,
         )
-        val newNumberOfEmployees = (frameworkDataAlpha.general!!.numberOfEmployees ?: BigDecimal.ZERO) + BigDecimal.ONE
+        val newNumberOfEmployees = (frameworkDataAlpha.general!!.numberOfEmployees!!.value ?: BigDecimal.ZERO) +
```
[the remark this answers is not in the document]
Comment (jakobkraus) [reply dataland-e2etests/src/test/kotlin/org/dataland/e2etests/tests/MetaDataControllerTest.kt]: Is marked as a change because I did not merge main into branch earlier

File: dataland-e2etests/src/test/kotlin/org/dataland/e2etests/tests/QaReportControllerTest.kt [hunk 21]
```diff
@@ -16,7 +16,7 @@ import org.junit.jupiter.api.Test
 import org.junit.jupiter.api.TestInstance
 import org.junit.jupiter.api.assertThrows
 import org.springframework.http.HttpStatus
-import java.util.*
+import org.dataland.datalandqaservice.openApiClient.model.EutaxonomyNonFinancialsData as EuTaxonomyNonFinancialsReport
```
[the remark this answers is not in the document]
Comment (jakobkraus) [reply dataland-e2etests/src/test/kotlin/org/dataland/e2etests/tests/QaReportControllerTest.kt]: This has been discussed (identical class name in backend)

File: dataland-e2etests/src/test/kotlin/org/dataland/e2etests/tests/QaReportControllerTest.kt
[the remark this answers is not in the document]
Comment (jakobkraus) [reply dataland-e2etests/src/test/kotlin/org/dataland/e2etests/tests/QaReportControllerTest.kt:1]: Generic private functions have been written

File: dataland-framework-toolbox/inputs/euTaxonomyNonFinancials/EutaxonomyNonFinancials.csv
Comment (jakobkraus) [reply dataland-framework-toolbox/inputs/euTaxonomyNonFinancials/EutaxonomyNonFinancials.csv:1]: Is marked as a change because I did not merge main into branch earlier

File: dataland-frontend/src/frameworks/esg-questionnaire/ViewConfig.ts
Comment (jakobkraus) [reply dataland-frontend/src/frameworks/esg-questionnaire/ViewConfig.ts:1]: Is marked as a change because I did not merge main into branch earlier

File: dataland-frontend/src/frameworks/eutaxonomy-non-financials/ViewConfig.ts
Comment (jakobkraus) [reply dataland-frontend/src/frameworks/eutaxonomy-non-financials/ViewConfig.ts:1]: Is marked as a change because I did not merge main into branch earlier

File: dataland-frontend/src/frameworks/lksg/ViewConfig.ts (hunk 9 above)
Comment (jakobkraus) [reply dataland-frontend/src/frameworks/lksg/ViewConfig.ts]: Is marked as a change because I did not merge main into branch earlier

File: dataland-internal-storage/src/main/kotlin/db/migration/V19__MigrateEutaxonomyNonFinancialsExtendedDatapoints.kt (hunk 11 above)
Comment (jakobkraus) [reply dataland-internal-storage/src/main/kotlin/db/migration/V19__MigrateEutaxonomyNonFinancialsExtendedDatapoints.kt]: Is marked as a change because I did not merge main into branch earlier

File: dataland-internal-storage/src/main/kotlin/db/migration/V19__MigrateEutaxonomyNonFinancialsExtendedDatapoints.kt (hunk 13 above)
Comment (jakobkraus) [reply dataland-internal-storage/src/main/kotlin/db/migration/V19__MigrateEutaxonomyNonFinancialsExtendedDatapoints.kt]: Is marked as a change because I did not merge main into branch earlier

File: dataland-e2etests/src/test/kotlin/org/dataland/e2etests/tests/QaReportControllerTest.kt [hunk 22]
```diff
@@ -26,70 +28,128 @@ class QaReportControllerTest {
...
+            .getCompanyInformationWithoutIdentifiers(1)
 
     @BeforeAll
     fun postTestDocuments() {
         documentManagerAccessor.uploadAllTestDocumentsAndAssurePersistence()
     }
 
-    private fun postSfdrQaReportForNewDataId(
-        sfdrQaReport: SfdrQaReport = SfdrQaReport(),
+    private fun <T>provideUploadInfo(
```
[the remark this answers is not in the document]
Comment (jakobkraus) [reply dataland-e2etests/src/test/kotlin/org/dataland/e2etests/tests/QaReportControllerTest.kt]: Function name has been changed according to suggestion
